packages/ncids-css/scss/components/nci-cta-strip/internal/_cta-strip.scss

```diff
@@ -20,20 +19,28 @@ ul.nci-cta-strip {
 	@include at-media-max('tablet-lg') {
 		@include u-display('block');
 	}
-	@include at-media('tablet-lg') {
-		padding:0 units('card');
+
+	@include at-media('mobile-lg') {
+		padding:0 units(10);
 	}
+
 	@include at-media('tablet') {
+		padding:0 units('card');
+	}
+
+	@include at-media('tablet-lg') {
 		padding:0 units(10);
 	}
+
 	@include at-media('desktop') {
-		padding:0 units(10);
+		padding:0 units(6);
 	}
+
 	@include at-media('widescreen') {
 		padding:0 units(6);
 	}
 
-	padding:0 0;
+	padding:0;
 	margin:0 auto;
```

Comment:
this formatting seems off, why is it mixed space after colon & no space after colon?  I would expect the space after colon.
e.g.
`padding: 0;` vs `padding:0;`